public/index.html

```diff
@@ -8,7 +8,7 @@
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="UniTandem provides higher education students in Finland with a chance to learn langauges and cultures through online tandem learning."
```

Comment:
This text looks good to me.

Reply:
Looks good. There is a typo on "languages" tho!

Reply:
It is okay

Reply:
Thank you!